aibolit/patterns/public_static_method/public_static_method.py

```diff
@@ -0,0 +1,21 @@
+import javalang
+
+
+class PublicStaticMethod:
+
+    def __init__(self):
+        pass
+
+    def __file_to_ast(self, filename: str) -> javalang.ast.Node:
+        """
+        Takes path to java class file and returns AST Tree
+        :param filename:
+        :return: Tree
+        """
+        with open(filename, encoding='utf-8') as file:
+            tree = javalang.parse.parse(file.read())
+        return tree
+
+    def value(self, filename: str):
+        tree = self.__file_to_ast(filename).filter(javalang.tree.MethodDeclaration)
+        return [node for path, node in tree if 'public' in node.modifiers and 'static' in node.modifiers]
```

Comment:
@paulodamaso You can actually do a little better:
`return [node for _, node in tree if all(elem in node.modifiers for elem in ['public', 'static'])]`